runtime/dotnet/core/Settings/BotFeatureSettings.cs

```diff
@@ -15,5 +15,8 @@ public class BotFeatureSettings
 
         // Use InspectionMiddleware
         public bool UseInspectionMiddleware { get; set; }
+
+        // Use RemoveRecipientMentionExtensions, only works in Teams channel
```

Comment:
I don't think it only works in Teams channel, is it?

Reply:
> I don't think it only works in Teams channel, is it?

Yes, according to the code, it doesn't depend on the channel type. Let me fix the comment.
```
        public static string RemoveMentionText(this IMessageActivity activity, string id)
        {
            foreach (var mention in activity.GetMentions().Where(mention => mention.Mentioned.Id == id))
            {
                if (mention.Text == null)
                {
                    activity.Text = Regex.Replace(activity.Text, "<at>" + Regex.Escape(mention.Mentioned.Name) + "</at>", string.Empty, RegexOptions.IgnoreCase).Trim();
                }
                else
                {
                   activity.Text = Regex.Replace(activity.Text, Regex.Escape(mention.Text), string.Empty, RegexOptions.IgnoreCase).Trim();
                }
            }

            return activity.Text;
        }
```